src/lib.rs

```diff
@@ -120,13 +120,19 @@ pub struct ParagraphInfo {
     ///
     /// <http://www.unicode.org/reports/tr9/#BD4>
     pub level: Level,
+
+    /// Whether any strong-RTL characters are present.
+    has_rtl: bool,
```

Comment:
this is a breaking change: Paragraph could previously be constructed manually (and often is, due to the lack of convenient APIs here)

Reply:
Ah, that's a pity; does that need to be the case, or is it worth reconsidering?

(Note that `./cargo test` didn't complain for me, so I guess that isn't explicitly tested at present.)

Reply:
I'd strongly like to avoid needing to do a semver breaking bump on a major dependency in the rust ecosystem.